docs/markdown/Python/python-integrations/awslambda-python.md

```diff
@@ -58,35 +67,34 @@ Pants will use [dependency inference](doc:targets) based on the `handler` field,
 You can optionally set the `output_path` field to change the generated zip file's path.
 
 > 🚧 Use `resource` instead of `file`
-> 
+>
 > `file` / `files` targets will not be included in the built AWS Lambda because filesystem APIs like `open()` would not load them as expected. Instead, use the `resource` and `resources` target. See [Assets and archives](doc:assets) for further explanation.
 
 Step 3: Run `package`
 ---------------------
 
-Now run `pants package` on your `python_awslambda` target to create a zipped file. 
+Now run `pants package` on your `python_awslambda` target to create a zipped file.
 
 For example:
 
 ```bash
-$ pants package project/awslambda_example.py
-Wrote code bundle to dist/project.zip
-  Runtime: python3.8
-  Handler: lambdex_handler.handler
+$ pants package project/:lambda
```

Comment:
The change from file path to proper target is #18737 , not related to the layout change.